crates/biome_html_formatter/tests/specs/html/text_expressions/expressions_line_boundaries.astro.snap

```diff
@@ -0,0 +1,58 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: text_expressions/expressions_line_boundaries.astro
+---
+# Input
+
+```astro
+<p>
+	Start {expr1}
+	{expr2} middle
+	end {expr3}
+</p>
+
+<div>{a}
+{b}</div>
+
+```
+
+
+=============================
+
+# Outputs
+
+## Output 1
+
+-----
+Indent style: Tab
+Indent width: 2
+Line ending: LF
+Line width: 80
+Attribute Position: Auto
+Bracket same line: false
+Whitespace sensitivity: css
+Indent script and style: false
+Self close void elements: never
+-----
+
+```astro
+<p>
+	Start {expr1}
+	{expr2}middle end {expr3}
```

Comment:
The space from the input `{expr2} middle` disappeared in the output.

It's a bit weird that it keeps the newline between `{expr1}` and `{expr2}`, but [it matches prettier behavior](https://biomejs.dev/playground/?lintRules=useSimplifiedLogicExpression&analyzerFixMode=safeAndUnsafeFixes&code=PABwAD4ACgAJAFMAdABhAHIAdAAgAHsAZQB4AHAAcgAxAH0ACgAJAHsAZQB4AHAAcgAyAH0AIABtAGkAZABkAGwAZQAKAAkAZQBuAGQAIAB7AGUAeABwAHIAMwB9AAoAPAAvAHAAPgAKAAoAPABkAGkAdgA%2BAHsAYQB9AAoAewBiAH0APAAvAGQAaQB2AD4A&language=svelte), so keep it.